### PR TITLE
Main

### PR DESCRIPTION
**Pull request by joannajiang112628**
readme file update

### CODE REVIEW

**Review by jorge-beauregard — COMMENTED** [review]
Should we add a comment saying that there's also an enum for system AND user assigned? I don't know if it's worth it though, just a thought.

File: sdk/communication/Azure.ResourceManager.Communication/samples/Sample1_ManagingCommunicationService.md [hunk 1]
```diff
@@ -47,6 +47,34 @@ ArmOperation<CommunicationServiceResource> communicationServiceLro = await colle
 CommunicationServiceResource communicationService = communicationServiceLro.Value;
 ```
 
+***Create a Communication Service with User Assigned Managed Identity***
+
+```C# Snippet:Managing_CommunicationService_CreateAnApplication with User Assigned Managed Identity
+var communicationService = await communicationManagementClient.CommunicationServices.CreateAsync(
+    resourceGroupName,
```
Comment (jorge-beauregard) [inline sdk/communication/Azure.ResourceManager.Communication/samples/Sample1_ManagingCommunicationService.md:54]: please declare the values of these variables just like the other examples. We want the clients to be able to copy and paste this and work out of the box

**Review by jorge-beauregard — COMMENTED** [review]
Please check how the original create example works and declares its variables. Make sure to follow the same structure. You'll see that example uses the resource group that was declared before to create a collection and they use the create or update async function.

File: sdk/communication/Azure.ResourceManager.Communication/samples/Sample1_ManagingCommunicationService.md [hunk 2]
```diff
@@ -47,6 +47,34 @@ ArmOperation<CommunicationServiceResource> communicationServiceLro = await colle
 CommunicationServiceResource communicationService = communicationServiceLro.Value;
 ```
 
+***Create a Communication Service with User Assigned Managed Identity***
+
+```C# Snippet:Managing_CommunicationService_CreateAnApplication with User Assigned Managed Identity
+var communicationService = await communicationManagementClient.CommunicationServices.CreateAsync(
```
Comment (jorge-beauregard) [inline sdk/communication/Azure.ResourceManager.Communication/samples/Sample1_ManagingCommunicationService.md:53]: Where did this communicationManagementClient come from?

Comment (joannajiang112628) [reply sdk/communication/Azure.ResourceManager.Communication/samples/Sample1_ManagingCommunicationService.md:53]: https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/create-communication-resource?tabs=windows&pivots=platform-net

File: sdk/communication/Azure.ResourceManager.Communication/samples/Sample1_ManagingCommunicationService.md [hunk 3]
```diff
@@ -47,6 +47,34 @@ ArmOperation<CommunicationServiceResource> communicationServiceLro = await colle
...
+    communicationServiceName,
+    new CommunicationServiceResource(
+        location: location,
+        dataLocation: dataLocation,
+        identity: new CommunicationServiceIdentity(type: IdentityType.UserAssigned, userAssignedIdentities: new Dictionary<string, CommunicationServiceUserAssignedIdentities>() { { userAssignedIdentityResourceId, new CommunicationServiceUserAssignedIdentities() } })
+    )
+);
+```
+
+***Create a Communication Service with User Assigned Managed Identity***
```
Comment (Joeleniqs) [inline sdk/communication/Azure.ResourceManager.Communication/samples/Sample1_ManagingCommunicationService.md:64]: This should read with System assigned

File: sdk/communication/Azure.ResourceManager.Communication/samples/Sample1_ManagingCommunicationService.md [hunk 4]
```diff
@@ -47,6 +47,34 @@ ArmOperation<CommunicationServiceResource> communicationServiceLro = await colle
 CommunicationServiceResource communicationService = communicationServiceLro.Value;
 ```
 
+***Create a Communication Service with User Assigned Managed Identity***
```
Comment (Joeleniqs) [inline sdk/communication/Azure.ResourceManager.Communication/samples/Sample1_ManagingCommunicationService.md:50]: Please confirm this code samples aren't valid

File: sdk/communication/Azure.ResourceManager.Communication/samples/Sample1_ManagingCommunicationService.md (hunk 2 above)
Comment (Joeleniqs) [inline sdk/communication/Azure.ResourceManager.Communication/samples/Sample1_ManagingCommunicationService.md:53]: it should be the communicationServiceResourceCollection Class for resource creation

Comment (jorge-beauregard) [reply sdk/communication/Azure.ResourceManager.Communication/samples/Sample1_ManagingCommunicationService.md:53]: Yes, but I mean in the code. You just put it without ever declaring it

Comment (jorge-beauregard) [reply sdk/communication/Azure.ResourceManager.Communication/samples/Sample1_ManagingCommunicationService.md:53]: Following the original example for resource creation should take you to the right place in regards to Joel's comment @joannajiang112628

File: sdk/communication/Azure.ResourceManager.Communication/samples/Sample1_ManagingCommunicationService.md [hunk 5]
```diff
@@ -47,6 +47,49 @@ ArmOperation<CommunicationServiceResource> communicationServiceLro = await colle
...
+***Create a Communication Service with System Assigned Managed Identity***
+
+```C# Snippet:Managing_CommunicationService_CreateAnApplication with System Assigned Managed Identityy
+string communicationServiceName = "myCommunicationService";
+CommunicationServiceResourceCollection collection = resourceGroup.GetCommunicationServiceResources();
+
+CommunicationServiceResourceData data = new CommunicationServiceResourceData("global")
+{
+    DataLocation = "UnitedStates",
+    Identity = new CommunicationServiceIdentity(type: IdentityType.SystemAssigned, userAssignedIdentities: null)
```
Comment (jorge-beauregard) [inline sdk/communication/Azure.ResourceManager.Communication/samples/Sample1_ManagingCommunicationService.md:85]: I don't think you need to send in the userAssignedIdentities if you are assigned a system identity

File: sdk/communication/Azure.ResourceManager.Communication/samples/Sample1_ManagingCommunicationService.md [hunk 6]
```diff
@@ -47,6 +47,49 @@ ArmOperation<CommunicationServiceResource> communicationServiceLro = await colle
...
+
+CommunicationServiceResourceData data = new CommunicationServiceResourceData("global")
+{
+    DataLocation = "UnitedStates",
+    Identity = identity
+};
+
+ArmOperation<CommunicationServiceResource> communicationServiceLro = await collection.CreateOrUpdateAsync(WaitUntil.Completed, communicationServiceName, data);
+CommunicationServiceResource communicationService = communicationServiceLro.Value;
+
```
Comment (jorge-beauregard) [inline sdk/communication/Azure.ResourceManager.Communication/samples/Sample1_ManagingCommunicationService.md:72]: nit: please remove this extra empty line. 1 empty line should be enough

**Pull request closed without merging**